pages/glossary/user-intervention.md

```diff
@@ -0,0 +1,9 @@
+---
+title: User intervention
+key: user-intervention
+---
+
+An action where a user operates a [user interface component][] of an [HTML document][].
```

Comment:
I don't think "action" and "operates" are clear enough.

Reply:
I removed "operates" from the definition. 
I couldn't find a way to write this without "action" (or a similar word). I decided to list the possible actions on HTML UI components (according to the HTML spec), which should be clear.